frontend/public/scss/home-page-video-component.scss

```diff
@@ -0,0 +1,107 @@
+@import "variables";
+@import "~bootstrap/scss/bootstrap";
+@import "~video.js/dist/video-js";
+
+.home-page-video-row {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+  @include media-breakpoint-down(sm) {
+      width: 100%;
+    }
+  .home-page-video-container {
+    margin-top: -60px;
+    width: 1440px;
+    max-width: 1440px;
+    height: 713px;
+    flex-shrink: 0;
+    display: flex;
+    .home-page-video-description {
+      width: 434px;
+      max-width: 434px;
+      display: flex;
+      flex-direction: column;
+      margin: 140px 23px 120px 60px;
+
+      h3 {
+        color: $home-page-header-blue;
+        font-size: 36px;
+        font-style: normal;
+        font-weight: 700;
+        line-height: 45px;
+        align-self: stretch;
+      }
+
+      p {
+        color: $home-page-header-blue;
+        font-size: 20px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 32px;
+        align-self: stretch;
+      }
+    }
+
+    .home-page-video-wrapper {
+      max-width: 923px;
+      width: 100%;
+      height: inherit;
+
+      .home-page-video-player {
+        width: 100%;
+        max-width: 923px;
+        height: inherit;
+        position: relative;
+        .vjs-default-skin{
+          width: 863px;
+          height: 485px;
+          position: absolute;
+          top: 120px;
+          right: 60px;
+          z-index: 99;
+
+        }
+        .video-js {
+      .vjs-big-play-button {
+        background: black;
+        opacity: 0.8;
+        width: 64px;
+        height: 64px;
+        border: none;
+        border-radius: 50%;
+
+        &:hover {
+          opacity: 0.6;
+        }
+      }
+
+      .vjs-poster {
+        height: fit-content;
+      }
+    }
+
+    .video-js .vjs-big-play-button .vjs-icon-placeholder:before {
+      top: 0.3em;
+    }
+
+    .vjs-youtube {
+
+      .vjs-big-play-button {
+        display: none;
+      }
+    }
+        .home-page-video-underlay {
```

Comment:
I am seeing this underlay background on all views
![image](https://github.com/mitodl/mitxonline/assets/3138890/dcfe6e49-3c9b-4c72-aa9d-dea5720e389e)